src/main/java/jenkins/advancedqueue/priority/strategy/PriorityJobProperty.java

```diff
@@ -119,7 +121,15 @@ public ListBoxModel getPriorities() {
 		public boolean isUsed(Job<?,?> owner) {
 			PriorityConfiguration configuration = PriorityConfiguration.get();
 			JobGroup jobGroup = configuration.getJobGroup(dummyCallback, owner);
-			return jobGroup != null;
+			if(jobGroup != null && jobGroup.isUsePriorityStrategies()) {
```

Comment:
The possibility to select a priority must only be shown if the job is indeed included in a JobGroup and that JobGroup supports the JobPropertyStrategy